usrsctplib/user_environment.c

```diff
@@ -222,7 +223,7 @@ read_random(void *buf, size_t size)
 
 	position = 0;
 	while (position < size) {
-		n = getrandom((char *)buf + position, size - position, 0);
+		n = syscall(__NR_getrandom, (char *)buf + position, size - position, 0);
```

Comment:
Can you put in a comment describing why you do the syscall directly. Then I'm happy to merge.

Reply:
Sorry, I saw this comment after I already pushed

Reply:
> Sorry, I saw this comment after I already pushed

No problem. As long as it works. We are only missing support for __native_client__. Any idea what can be used there?

Reply:
I don't know much about native client, but I found `nacl_secure_random`: https://source.chromium.org/chromium/chromium/src/+/master:base/rand_util_nacl.cc

Reply:
OK. I can use that and we will see if some reports a bug. I have no way to test it...